buildSrc/src/main/kotlin/com/datadog/gradle/plugin/benchmark/ReviewBenchmarkResultsTask.kt

```diff
@@ -45,12 +53,51 @@ open class ReviewBenchmarkResultsTask : DefaultTask() {
 
     // region Internal
 
-    private fun processDeviceBenchmarks(deviceDir: File) {
+    private fun verifyRelativeThresholds(benchmarksResults: MutableMap<String, Long>) {
+        extension.relativeThresholds.forEach {
+            val testAName = it.betweenTest
+            val testBName = it.andTest
+            val testAExecTime = benchmarksResults[testAName]
+            val testBExecTime = benchmarksResults[testBName]
+
+            checkNotNull(testAExecTime) {
+                missingBenchmarkTestMessage(it, testAName)
+            }
+            checkNotNull(testBExecTime) {
+                missingBenchmarkTestMessage(it, testBName)
+            }
+
+            val difference = TimeUnit.NANOSECONDS.toSeconds(abs(testAExecTime - testBExecTime))
```

Comment:
Does this mean you expect the difference in Relative benchmark to always be in seconds ? This is lacking a lot of precision I think. Even though you don't need that much info for a 5 second difference in your current case, it'd be better to use consistent units across the same plugin.

Reply:
oh my bad here should have been millis